rever.xsh

```diff
@@ -37,12 +37,18 @@ git diff --cached --exit-code
 
 $PROJECT = 'svgdigitizer'
 
+from rever.activities.command import command
+
+command('build', 'python -m build')
+command('twine', 'twine upload dist/*')
```

Comment:
If you can predict the name of the file in `dist/` it's better to use it instead of `dist/*`. Otherwise, you might be uploading some weird things to PyPI.

Reply:
What do you mean by predicting the name? Such as:
`'svgdigitizer-' + $VERSION + '.tar.gz'`?

Honestly, I simply took these lines from your suggested reference. :)

Reply:
Yes, something like that.